initial_positions:
  elbow: 1.7
  shoulder_pan: 0.0
  shoulder_tilt: -0.75
  wrist_pitch: 0.0
  wrist_roll: 0.0
